CorsixTH/Lua/world.lua

```diff
@@ -984,35 +977,30 @@ function World:onTick()
       if not lfs.attributes(dir .. "Autosaves", "modification") then
         lfs.mkdir(dir .. "Autosaves")
       end
-      local status, err = pcall(TheApp.save, TheApp, dir .. "Autosaves" .. pathsep .. "Autosave" .. self.month .. ".sav")
+      local status, err = pcall(TheApp.save, TheApp, dir .. "Autosaves" .. pathsep .. "Autosave" .. self.game_date:monthOfYear() .. ".sav")
       if not status then
         print("Error while autosaving game: " .. err)
       end
     end
-    if self.year == 1 and self.month == 1 and self.day == 1 and self.hour == 0 then
-      if not self.ui.start_tutorial then
-        self.ui:addWindow(UIWatch(self.ui, "initial_opening"))
-        self.ui:showBriefing()
-      end
+    if self.game_date == Date() and not self.ui.start_tutorial then
```

Comment:
Technically correct, but Date(1) might be clearer? I'll leave that up to you.

Reply:
I would say either `Date()` or `Date(1,1,1,0)`. Defining some of the parameters and relying on the defaults for other ones seems a bit more confusing to me. But yeah I prefer the default solution.